### PR TITLE
chore: new PR template

### LINKED ISSUE #52: chore: create a new PR template
### Description

Create a PR template. 

1. Add 1 comment in its topmost part
```
   If you need help with you PR or if you want to to discuss something with us, join the WalkingPal [Discord community](https://discord.gg/Et3aQTsW6a)!
```
3. Add this checkbox shown below in the template's textarea (not commented)
- [ ] I have read and I am following [the guidelines for creating a new Pull Request](https://github.com/WalkingPal/walking-pal-web/blob/dev/.docs/CONTRIBUTING.md#pull-requests-review-criteria)

4. Add 1 comment in its bottom most part
```
   Link this PR to your issues by writing: fixes #issue_id OR closes #issue_id
```

### PR DESCRIPTION
**Pull request by tanmay-S12**
pls review the PR and do consider merging it, if it's correct. 

closes #52

### CODE REVIEW

File: .github/pull_request_template.md/pull_request1_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!-->
+* ### If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!
+
+   ---------------------------------------------------
+
+
+   ### Checklist :
```
Comment (ashuvssut) [inline .github/pull_request_template.md/pull_request1_template.md:7]: 1. In previous PR, there was just those 6 lines only. But, in this PR there is again excess lines. Please put those 6 lines back. Remove all this

2. Your folder name is "pull_request_template.md". It must be PULL_REQUEST_TEMPLATE

**Review by ashuvssut — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md/pull_request1_template.md (hunk 1 above)
Comment (tanmay-S12) [reply .github/pull_request_template.md/pull_request1_template.md]: > 1. In previous PR, there was just those 6 lines only. But, in this PR there is again excess lines. Please put those 6 lines back. Remove all this
> 2. Your folder name is "pull_request_template.md". It must be PULL_REQUEST_TEMPLATE

made the changes. pls check

Comment (tanmay-S12) [reply .github/pull_request_template.md/pull_request1_template.md]: @ashuvssut Hey, Thanks for guiding me through this and for your support. It was great learning. 
Will hactoberfest consider this PR for me?

Comment (ashuvssut) [reply .github/pull_request_template.md/pull_request1_template.md]: yes

**Pull request merged**